src/mavsdk_server/tools/package_mavsdk_server_framework.bash

```diff
@@ -37,6 +40,12 @@ xcrun codesign --verbose --sign - ${IOS_SIM_FAT_BACKEND_DIR}/mavsdk_server.frame
 echo "Running lipo for macos"
 mkdir -p ${MACOS_FAT_BACKEND_DIR}
 cp -a ${MACOS_X64_BACKEND_DIR}/mavsdk_server.framework ${MACOS_FAT_BACKEND_DIR}
+# Fix up symlinks that seem to break when github artifacts unzips the archives
```

Comment:
Oh, I think I have seen this in the past... but I can't remember where (there are no other symlinks in this script, so probably somewhere else :thinking:).

Good catch!